README.md

```diff
@@ -144,18 +144,18 @@ exposed via reStructured Text extension.
 ### Markdown
 
 Nikola follows the [syntax
-rules](http://daringfireball.net/projects/markdown/syntax) of the original
+rules](https://daringfireball.net/projects/markdown/syntax) of the original
 `markdown.pl` with the following
-[extensions](http://pythonhosted.org/Markdown/extensions/index.html) enabled:
+[extensions](https://pythonhosted.org/Markdown/extensions/index.html) enabled:
```

Comment:
http://pythonhosted.org/Markdown/extensions/index.html redirects to https://pythonhosted.org/Markdown/extensions/index.html and it doesn't work, it seems to be a dead link

Reply:
I will use https://pypi.org/

Reply:
Done